packages/babel-plugin-minify-constant-folding/src/replacements.js

```diff
@@ -43,11 +43,24 @@ module.exports = ({ types: t }) => {
           let bad = false;
           const str = this.elements
             .map(el => {
-              if (!t.isLiteral(el)) {
-                bad = true;
-                return;
+              if (t.isRegExpLiteral(el)) {
+                return `/${el.pattern}/${el.flags}`;
               }
-              return el.value;
+              if (t.isNullLiteral(el)) {
+                return null;
+              }
+              if (
+                t.isStringLiteral(el) ||
+                t.isBooleanLiteral(el) ||
+                t.isNumericLiteral(el)
+              ) {
+                return el.value;
+              }
+              if (t.isTemplateLiteral(el) && el.expressions.length === 0) {
+                return el.quasis[0].value.cooked;
```

Comment:
check this https://github.com/babel/babel/blob/7.0/packages/babel-traverse/src/path/evaluation.js#L391